frames/section.lua

```diff
@@ -222,6 +234,31 @@ local function onTitleClickOrDrop(section)
   events:SendMessage('bags/FullRefreshAll')
 end
 
+---@param section Section
+local function onTitleRightClick(section) 
+  if section.headerDisabled then
+    return
+  end
+  
+  if (addon.atBank == false) then return end
+  
+  -- CellData[] { bagid, slotid, itemInfo, etc }
+  local items = {}
+
+  for _, cell in pairs(section.content.cells) do
+    if not cell.data.isItemEmpty then
+      table.insert(items, cell.data)
+    end
+  end
+  
+  async:Do(function()
+    for _, item in pairs(items) do
+      C_Container.UseContainerItem(item.bagid, item.slotid)
```

Comment:
One thing we need to consider checking here is the state of the bag. Because this function can happen N frames into the future, how do we ensure the item is still the same item?

Reply:
Before the async starts i'm getting the item list. I can check inside if if the itemID in that bag/slot is still the same... you think that would be enough?

Reply:
Yes, let's try that and see what the performance is like. You'll also need to deep copy the item data instead of inserting it's reference into a list.